src/test/java/org/primeframework/mvc/GlobalTest.java

```diff
@@ -85,6 +87,12 @@ public void beforeClass() {
     jsonDir = Path.of("src/test/resources/json");
   }
 
+  @BeforeMethod
+  public void beforeMethod() {
+    super.beforeMethod();
+    FullFormAction.reset();
```

Comment:
Should this just go in `FullFormAction.java`? Does it need to be here?

Reply:
Sure (fixed that). I was unsure about whether to echo this back out in the FTL and test that way or do this. I like this way slightly better since I'm mainly trying to ensure things get to the Java action class correctly for this case, but I'm open to changing it.